Objective-C/CBLCollection.mm

```diff
@@ -52,6 +58,9 @@ - (instancetype) initWithDB: (CBLDatabase*)db
         _name = slice2string(spec.name);
         _scope = [[CBLScope alloc] initWithDB: db name: slice2string(spec.scope)];
         
+        NSString* qName = $sprintf(@"Collection <%@>", _name);
```

Comment:
Will it be OK if I have two instances of the same collection?

Reply:
yes. I will update to include the self address as well. 

I copied this code from database, which means the same issue(non unique GCD queue can be created) can happen in Database as well .